src/test/java/ca/team2706/frc/robot/subsystems/SensorExtrasTest.java

```diff
@@ -0,0 +1,86 @@
+package ca.team2706.frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import edu.wpi.first.wpilibj.*;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import mockit.Expectations;
+import mockit.Mocked;
+import mockit.Tested;
+import mockit.Verifications;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.lang.reflect.Field;
+
+import static org.junit.Assert.assertEquals;
+
+public class SensorExtrasTest {
+
+    @Tested
+    private SensorExtras sensorExtras;
+
+    @Mocked
+    private WPI_TalonSRX talon;
+
+    @Mocked
+    private PWM pwm;
+
+    @Mocked
+    private AnalogInput analogInput;
+
+    @Mocked
+    private DigitalInput dio;
+
+    @Mocked
+    private Relay relay;
+
+    @Mocked
+    Subsystem subsystem;
```

Comment:
I see why it'd be useful to mock subsystem in order to verify allocation later, but the thing is it's not actually a dependency, it's part of the codebase. If there's a way around mocking this that'd be great.